### ISSUE TITLE
Physical navigation buttons keep vibrating when screen off

### ISSUE THREAD

**Issue by delold** [AChep/AcDisplay#84]
It seems like the navigation buttons aren't turned off when the screen is off. It's a little bit annoying, as the phone may start vibrating when in a pocket. 

Running on HTC Desire X, Android 4.1 (AcDisplay is in compatibility mode).

**Comment by SMillerDev** [comment]
Similar to #65, maybe @AChep can merge the issues?

**Comment by AChep** [comment]
@SMillerDev nah, I don't think they are related. Anyway, @delold I need more info about your setup.

**Comment by delold** [comment]
@AChep sure
 - Lock screen *on*
     - Allowed even without notifications 
 - Active mode *off*
 - Wakeup on incoming notification *off*
 - Inactive time - *off*
 - Double tap to sleep - *off*

The app is running on background, as there is a running service (in the notification tray). 

Turning off the AcDisplay "fixes" the issue
